corehq/apps/products/fixtures.py

```diff
@@ -1,17 +1,47 @@
 from corehq.apps.products.models import Product
 from corehq.apps.commtrack.fixtures import _simple_fixture_generator
+from corehq.apps.products.models import SQLProduct
+from corehq.apps.custom_data_fields.models import CustomDataFieldsDefinition
+
+PRODUCT_FIELDS = [
+    'name',
+    'unit',
+    'code',
+    'description',
+    'category',
+    'program_id',
+    'cost',
+    'product_data'
+]
+
+CUSTOM_DATA_SLUG = 'product_data'
+
+
+def product_fixture_generator_json(domain):
+    if SQLProduct.objects.filter(domain=domain).count() == 0:
```

Comment:
might want to use `.exists()` here. not 100% true whether it matters for performance, but even for readability is a small improvement